cmd/terramate/cli/cli.go

```diff
@@ -946,7 +946,7 @@ func generateDot(
 		Logger()
 
 	parent := dotGraph.Node(getLabel(stackval))
-	for _, childid := range graph.ChildrenOf(id) {
```

Comment:
For general graphs, it really depend on how you look at them, but for directed graphs, the dependency arrow is explicit and calling this method "ChildrenOf" was wrong, it actually return the parentsOf/ancestorsOf... The change has no other implication in the code.